Makefile

```diff
@@ -51,15 +50,14 @@ test: ## Run tests
 # build packages
 
 build: ## Build a release (sdist and wheel)
-	$(python) -m pip install --upgrade setuptools wheel
-	$(python) setup.py sdist bdist_wheel
+	$(python) -m pip install --upgrade build
+	$(python) -m build
 .PHONY: build
 
 test-release: build ## Test a built release
 	./test-release
 .PHONY: test-release
 
 release: ## Upload a built release
-	$(twine) upload dist/colorama-$(version)*
+	$(twine) upload dist/colorama-*{.whl,.tar.gz}
```

Comment:
We encountered an issue in our last release where the globs using `{}` didn't work in the Makefile. We had to remove them.